README.md

```diff
@@ -212,6 +221,64 @@ docker run -p 6379:6379 envirodgi/db-import-worker -e <ENVIRONMENT VARIABLES> .
+
+    *(Note: because of technical issues around imported data, we often store more versions than we should according to the above definition [e.g. we might still have a record for Tuesday]. Versions have a `different` field that indicates whether a version is different from the previous one, and the API only returns versions that are `different` unless you explicitly request otherwise.)*
+
+- **Annotations**, which represent an analysis about what’s changed between any two *versions* of a *page*. Annotations have a specialized `priority` and `significance`, which are numbers between 0 and 1, an `author`, indicating who made the analysis (it could be a bot account), and an `annotation` field, which is a JSON object with no specified structure (inside this field, annotations can include any data desired).
+
+There are several other kinds of objects, but they are subservient to the ones above:
+
+- **Changes**, which serve to connect any two *versions* of a *page*. *Annotations* are actually connected to *changes*, rather than directly to two *versions*. You can also generate diffs for a given *change*.
+
+- **Tags**, which can be applied to pages. They help sort and categorize things. Most tags are manually applied, but the application auto-generates a few:
+    - `domain:<domain name>`, e.g. `domain:www.epa.gov` for a page at `https://www.epa.gov/citizen-science`
+    - `2l-domain:<second-level domain name>` e.g. `2l-domain:epa.gov` for a page at `https://www.epa.gov/citizen-science`
+
+- **Maintainers**, which can be applied to pages. They represent organizations that maintain a given page. For example, the page at `https://www.epa.gov/citizen-science` is maintained by `EPA`.
+
+- **Imports** model requests to import new data and the results of the import operation.
+
+- **Users** model people (both human and bots) who can view, import, and annotate data. You currently have to have a user account to do anything in the application, though we hope accounts will not be needed to view public data in the future.
+
+Actual database schemas for each of these tables is listed in [`db/schema.rb`](./db/schema.rb).
+
+
+### How Data Gets Loaded
+
+The web-monitoring-db project does not actually monitor or scrape pages on the web. Instead, we rely on importing data from other services, like [the Internet Archive](https://archive.org). Each day, a script queries other services for historical snapshots and sends the results to the `/api/v0/imports` endpoint.
+
+Most of the data sent to `/api/v0/imports` matches up directly with the structure of the [`Version` model](./db/schema.rb). However, the `uri` field in an import is treated specially.
```

Comment:
`Version` model was linking to `.../models/version.rb` before, but now points to `schema.rb`. It makes it clearer (for me at least!) what the table structure of `versions` is by looking at `schema.rb`, but not sure if `version.rb` is more useful for experienced Rails developers. Just double-checking this change was intentional

Reply:
> not sure if version.rb is more useful for experienced Rails developers.

It’s not! :)

I changed it because I realized while updating that this is all about what data is available from an object, and ActiveRecord classes don’t give you any of that information. It’s all in `schema.rb`. (The classes have extra methods and validations and so on, but the actual data properties are determined dynamically at runtime when data is loaded from a database query.)